src/data/routes.ts

```diff
@@ -7,71 +7,92 @@ export type RouteInfo = {
 
 export const gettingStarted = [
   {
-    route: "/getting_started/using_the_cli",
-    title: "Using the CLI",
+    route: '/getting_started/using_the_cli',
```

Comment:
Can you rollback these changes from `"` to `'` ?
These might be due to a formatter configured in vscode that format files on save or something similar.